sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/theme/AppColors.kt

```diff
@@ -0,0 +1,29 @@
+package co.nimblehq.sample.compose.ui.theme
+
+import androidx.compose.material.*
+import androidx.compose.runtime.staticCompositionLocalOf
+
+/**
+ * Extend final [Colors] class to provide more custom app colors.
+ */
+data class AppColors(
+    val themeColors: Colors,
+
+    // Custom colors here
+)
+
+internal val LightColorPalette = AppColors(
+    themeColors = lightColors(
+        primary = GreenCitrus,
+        secondary = GreenChristi
```

Comment:
We don't have the example for the `UI` that was applied `secondary` yet, e.g., `Floating action buttons`, So I think we can remove `secondary` on both `sample/template-compose`. What do you think? 🙏 
```suggestion
```

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/376/commits/33274a5d714f980bd6d0c95316bfe07e938f3351